source/includes/aggregation.php

```diff
@@ -0,0 +1,43 @@
+<?php
+require 'vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+$db = $client->sample_restaurants;
```

Comment:
If you end up using `Collection::explain()`, there will be no need to assign a separate `$db` variable.